### PR TITLE
Introduce action items api

### PR DESCRIPTION
**Pull request by tuykin**
### Description

Introduce API for action items:

Снимок экрана 2020-07-09 в 15 04 20
Снимок экрана 2020-07-09 в 15 04 48
Снимок экрана 2020-07-09 в 15 07 07
Снимок экрана 2020-07-09 в 15 07 36

### CODE REVIEW

File: app/graphql/types/board_type.rb [hunk 1]
```diff
@@ -11,6 +11,10 @@ class BoardType < Types::BaseObject
     field :show_card_author, Boolean, null: false
     field :status, String, null:  true
     field :reversed, Boolean, null: false
+
+    field :action_items, [ActionItemType], null: false do
+      description 'List of Action Items'
+    end
```
Comment (akaoss) [inline app/graphql/types/board_type.rb:17]: Ok, let's expose `action_items` inside `board`.
But according to the FE we'll display action items inside cards.
So please also add `field :cards` inside `board` and add `action_items` inside `card` too.

File: app/graphql/mutations/create_action_item.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Mutations
+  class CreateActionItem < Mutations::BaseMutation
+    argument :text, String, required: true
+    argument :card_id, ID, required: true
+    argument :assignee_id, String, required: false
+    argument :done, Boolean, required: false
+
+    field :action_item, Types::ActionItemType, null: true
+
+    def resolve(args)
```
Comment (akaoss) [inline app/graphql/mutations/create_action_item.rb:10]: Please, create a simple business action in `/actions/action_item/create.rb` and create ActiveRecord there.

File: app/graphql/types/board_type.rb [hunk 3]
```diff
@@ -12,5 +12,7 @@ class BoardType < Types::BaseObject
     field :status, String, null:  true
     field :reversed, Boolean, null: false
     field :columns, [ColumnType], null: false
+    field :cards, [CardType], null: false
+    field :action_items, [ActionItemType], null: false
```
Comment (tuykin) [inline app/graphql/types/board_type.rb:16]: @jalapl you mean this one?

Comment (jalapl) [reply app/graphql/types/board_type.rb:16]: yes ;)

**Pull request merged**